frontend/src/Button/Button.tsx

```diff
@@ -11,21 +11,22 @@ export enum ButtonOptions {
 type ButtonProps = {
   children?: string;
   type: ButtonOptions;
+  className?: string;
 };
 
 function Button(props: ButtonProps) {
   return (
     <button className={classes.button}>
       {props.type === ButtonOptions.Plus && (
-        <PlusIcon className={classes.icon}></PlusIcon>
+        <PlusIcon className={`${classes.icon} ${props.className}`}></PlusIcon>
```

Comment:
note that this will pass in a class named 'undefined' if `props.className` is undefined. consider changing to:
`...${props.className? props.className : ''}`
and please apply this fix to other similar instances below :)